satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -82,429 +82,494 @@ datasets:
     resolution: 2000
     file_type: nc_fci_clm
     file_key: cloud_state
-    long_name: cloud_mask_classification
+    standard_name: cloud_mask_classification
+    fill_value: -127
+    import_enum_information: True
 
   quality_illumination_clm:
     name: quality_illumination_clm
     resolution: 2000
     file_type: nc_fci_clm
     file_key: quality_illumination
-    long_name: illumination_classification
+    standard_name: status_flag
+    fill_value: -127
+    import_enum_information: True
 
   quality_nwp_parameters_clm:
     name: quality_nwp_parameters_clm
     resolution: 2000
     file_type: nc_fci_clm
     file_key: quality_nwp_parameters
-    long_name: quality_index
+    standard_name: status_flag
+    fill_value: -127
+    import_enum_information: True
 
   quality_MTG_parameters_clm:
```

Comment:
dataset names are typically all lower case, please rename to quality_mtg_parameters (also remove `_clm`, see other comment)